source/tests/pt/modifier/test_dipole_charge.py

```
@@ -0,0 +1,220 @@
+            to_numpy_array(ret0["energy"]), to_numpy_array(ret1["energy"])
+        )
+        np.testing.assert_allclose(
+            to_numpy_array(ret0["force"]), to_numpy_array(ret1["force"])
+        )
+        np.testing.assert_allclose(
+            to_numpy_array(ret0["virial"]), to_numpy_array(ret1["virial"])
+        )
+
+    def test_box_none_warning(self):
+        """Test that a RuntimeWarning is raised when box is None."""
+        coord, box, atype = ref_data()
```

Ruff (0.14.10)

Unpacked variable `box` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)